db/structure.sql

```diff
@@ -2,20 +2,32 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 11.5
--- Dumped by pg_dump version 11.5
+-- Dumped from database version 9.5.14
+-- Dumped by pg_dump version 9.5.14
```

Comment:
ping @auspicacious on the state of api psql DB version. Pretty sure it's not on 11 yet.